sale_cancel_confirmed_invoice/models/sale_order.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def _show_cancel_wizard(self):
+        for order in self:
+            if (
+                not order._context.get("disable_cancel_warning")
```

Comment:
```suggestion
                not order.env.context.get("disable_cancel_warning")
```

_context key is deprecated since a long time

Reply:
Updated to remove the deprecated! Thanks!